README.md

```diff
@@ -1,369 +1,168 @@
 
-* `git clone https://github.com/poanetwork/poa-bridge.git`
-* `cd poa-bridge`
-* Run:
+5. Install and run the POA Bridge Oracle.
+  * Got to the `sokol-kovan-bridge` folder and  `git clone https://github.com/poanetwork/token-bridge`
+  * Follow instructions in the [POA NodeJS Oracle repo](https://github.com/poanetwork/token-bridge).
 
-```
-make
-```
-7. Go back to directory up.
-`cd ..`
-8. Create `config.toml` file  
-`nano config.toml`  
-Replace
-`0xETH_ACCOUNT_VALIDATOR_SOKOL` to your `HOME_VALIDATORS` address   
-`0xETH_ACCOUNT_VALIDATOR_KOVAN` to your `FOREIGN_VALIDATORS` address  
-and `authorities` to your `VALIDATORS` from .env deploy.js above
-
-```yaml
-keystore = "keys"
-
-[home]
-account = "0xETH_ACCOUNT_VALIDATOR_SOKOL"
-required_confirmations = 0
-rpc_host = "https://sokol.poa.network"
-rpc_port = 443
-password = "password.txt"
-
-[foreign]
-account = "0xETH_ACCOUNT_VALIDATOR_KOVAN"
-required_confirmations = 0
-rpc_host = "https://kovan.infura.io/mew"
-rpc_port = 443
-password = "password.txt"
-
-[authorities]
-accounts = [
-  "0x..",
-  "0x.."
-]
-required_signatures = 1
-
-[transactions]
-deposit_relay = { gas = 3000000, gas_price = 1000000000 }
-withdraw_relay = { gas = 3000000, gas_price = 1000000000 }
-withdraw_confirm = { gas = 3000000, gas_price = 1000000000 }
-```
-9. Create db.toml file  
-`nano db.toml`  
-Insert addresses from  
-`cat poa-bridge-contracts/deploy/bridgeDeploymentResults.json`  
-```yaml
-home_contract_address = "0xE46...E0a" #YOUR HOME CONTRACT ADDRESS THAT YOU DEPLOYED IN STEP#3 
-foreign_contract_address = "0x90...3eA" #YOUR FOREIGN CONTRACT ADDRESS THAT YOU DEPLOYED IN STEP#4
-checked_deposit_relay = 1400663 # last checked deposits events on Home network
-checked_withdraw_relay = 6342830 # last checked withdraw events on Foreign network
-checked_withdraw_confirm = 6342830 # last checked withdraw events on Foreign network
-```
-Exit `nano` by Ctrl-O and Ctrl-X  
 
-* mkdir `keys`
-* put validator JSON keystore into `keys` folder
-* create `password.txt` file and insert your password for the key
+If successful, you will see bridge processes run when you issue a command. For example run `npm run watcher:signature-request`
 
-10. At this step your folder structure should look like this
+**Example NPM Output:**
 ```bash
-.
-└── sokol-kovan-bridge
-    ├── config.toml
-    ├── db.toml
-    ├── poa-bridge
-    └── poa-bridge-contracts
-```
-11. Run the bridge app:
-```
-env RUST_LOG=info ./poa-bridge/target/release/bridge --config config.toml --database db.toml
+[1539195000507] INFO (watcher-signature-request): Connected to redis
+[1539195000545] INFO (watcher-signature-request): Connected to amqp Broker
+[1539195006085] INFO (watcher-signature-request): Found 0 UserRequestForSignature events
+[1539195011467] INFO (watcher-signature-request): Found 0 UserRequestForSignature events
 ```
 
-If everything went well, you should see some logs from the  bridge:
+**Example Docker Output:**
+
 ```bash
```

Comment:
I would say that you will see this depending on configuration. In my setup I have never seen these messages and I needed to access the logs in the container explicitly.